_slides/git.html

```diff
@@ -84,25 +84,41 @@
 </svg>
 
 <section>
-	<h1>Devenir efficace avec Git</h1>
+	<h1>Becoming efficient with Git</h1>
```

Comment:
Are we sure about "efficient" here vs "proficient"?

Reply:
Right, "proficient" seems better.